src/superagent.js

```diff
@@ -1,6 +1,6 @@
-import assert from './assert';
-import Action from './Action';
-import Entity from './index';
+import Action from './Action.js';
+import assert from './assert.js';
+import Entity from './Entity.js';
```

Comment:
What's this superagent stuff used for? I spent a bunch of time removing superagent from a lot of our stuff a couple months ago, switching it to just use native `fetch` instead. Maybe this can be removed?

Reply:
Ok cool... yeah as long as the places relying on it aren't part of BSI, then they can stay back on the old version until they can update.

Reply:
As I'm quite out of the loop on current BSI, [mind quickly sanity checking](https://search.d2l.dev/search?project=Brightspace&full=%22siren-parser%2Fsuperagent%22+OR+%22siren-parser%2Fsrc%2Fsuperagent%22&defs=&refs=&path=&hist=&type=&xrd=&nn=1&si=path&si=path)? As far as I can tell, that's all either standalone things and/or old stuff that I presume isn't in BSI. But in all honesty, I have completely forgotten where to check what is included in BSI 😬

Reply:
Those all look safe to me, they're mostly all FRAs and none of them are in BSI's package-lock. Thanks!